### PR TITLE
Done

### PR DESCRIPTION
**Pull request by dorooleg**
Done

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
+
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".MainActivity"
+            android:configChanges="keyboardHidden|orientation|screenSize">
```
Comment (thevery) [inline app/src/main/AndroidManifest.xml:15]: configChanges and landscape layout, really?

Comment (dorooleg) [inline app/src/main/AndroidManifest.xml:15]: Oh my God, it's really not working)) Thanks! I am readed that use configChanges is bad practice. Is it really?

Comment (thevery) [inline app/src/main/AndroidManifest.xml:15]: yes it is

File: app/src/main/res/layout/main.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        android:text="@string/my_full_name"/>
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:id="@+id/my_photo_image_view" />
+
+
+</LinearLayout>
```
Comment (thevery) [inline app/src/main/res/layout/main.xml:22]: Newline should not be removed

File: app/src/main/java/com/csc/lesson1/MainActivity.java [hunk 3]
```diff
@@ -1,12 +1,44 @@
...
 import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.ImageView;
+
+import com.squareup.picasso.Picasso;
 
 public class MainActivity extends AppCompatActivity {
 
+    final String ADDRESS_TO_MY_IMAGE = "https://pp.vk.me/c625827/v625827693/13487/q4ALHbxcvME.jpg";
```
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:15]: make it private as well

File: app/src/main/java/com/csc/lesson1/MainActivity.java [hunk 4]
```diff
@@ -1,12 +1,44 @@
...
+                .load(ADDRESS_TO_MY_IMAGE)
+                .placeholder(R.drawable.error_load_image)
+                .error(R.drawable.error_load_image)
+                .fit()
+                .into(myPhotoImageView);
+
+        myPhotoImageView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Picasso.with(ptrOnMainActivity)
```
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:33]: `v.getContext()`

File: app/src/main/java/com/csc/lesson1/MainActivity.java [hunk 5]
```diff
@@ -1,12 +1,44 @@
...
+                        .load(ADDRESS_TO_MY_IMAGE)
+                        .placeholder(R.drawable.error_load_image)
```
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:35]: placeholder and error should be different

File: app/src/main/java/com/csc/lesson1/MainActivity.java (hunk 4 above)
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:33]: copy-paste!

File: app/src/main/java/com/csc/lesson1/MainActivity.java [hunk 6]
```diff
@@ -1,12 +1,44 @@
...
+                        .error(R.drawable.error_load_image)
+                        .fit()
+                        .into(myPhotoImageView);
+            }
+        });
+
+        Log.d("BaseEvent", "onCreate call");
```
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:42]: unconditional log

File: app/src/main/res/layout/main.xml [hunk 7]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (thevery) [inline app/src/main/res/layout/main.xml:1]: XML should be reformatted

**Comment by thevery** [comment]
+:
- Localization
- Caching
- App name
- Placeholder (though not perfect)

-:
- Photo is stretched
- Useless `android:configChanges`
- Copy-paste
- Logs

Total:
10 + 3.5 - 2.5 = 11

**Pull request closed without merging**